### PR TITLE
chore(demo): new homepage

### PR DESCRIPTION
**Pull request by benouat**
Here is a new version of the demo site homepage.

Main changes are:

* A new svg has been created.
* The favicon and logo have been regenerated based on the new svg.
* The navbar is now also inside a container to be consistent, no more full width. The logo has been removed.
* Regarding social buttons for Github, only star button is useful and make sense. All the other one have been removed.
* Big, full screen and centred logo.
* The jumbotron gradient has been rotated (it now has a 135deg angle)
* The package version number has been added (it is dynamically extracted from `package.json` and then publish to typescript via webpack DefinePlugin.

Here is a screenshot to have a overview without the need to clone and build.

screenshot

### CODE REVIEW

**Comment by pkozlowski-opensource** [comment]
I like it a lot @benouat - thnx for looking into it! 

There are just 2 cosmetic changes I would like to discuss:
* we should probably rephrase text in the jumbotron as we are repeating "Bootstrap" 2 times
* my first reaction was that B in the logo is a bit too bold, but maybe I'm too used to the previous one; let me look at it a bit more

Once again, thnx for this PR!

**Comment by benouat** [comment]
@pkozlowski-opensource The repeated "bootstrap" is a totally valid point !  What about

> Bootstrap 4 widgets/components, powered by Angular 2 and written by the angular-ui team.

I also realised that I missed your first point in #521, the first link being external.

I have an idea to add 2 more buttons to the jumbotron. Something like that

screen shot 2016-08-12 at 16 41 29


WDYT ?

**Comment by pkozlowski-opensource** [comment]
I like the buttons! Did you try their smaller version?

**Comment by benouat** [comment]
yes I tried it. It felt too small.

screen shot 2016-08-12 at 16 53 04

screen shot 2016-08-12 at 16 52 34

**Comment by wesleycho** [comment]
The jumbotron section needs vertical alignment IMO.

IMO it should be smaller too, but I was overruled on that, so otherwise I don't have much to add.

**Comment by benouat** [comment]
@pkozlowski-opensource @wesleycho I took your comments into considerations and pushed a new commit with these fixes.

Here the result (I highlighted one button to show you the hover effect)

screen shot 2016-08-12 at 17 20 25

**Comment by icfantv** [comment]
Please see my comments in your other PR thread.

**Comment by benouat** [comment]
@icfantv here is a global comment from my other [PR #593](https://github.com/ng-bootstrap/ng-bootstrap/pull/593#issuecomment-240018316)

> @icfantv Thank's for your comment ! Just for you to know, I am not going into the wild without an guidance here... I am sitting in the same room as @pkozlowski-opensource and have been discussing these changes a lot with him.

File: demo/src/app/default/default.component.html [hunk 1]
```diff
@@ -1,26 +1,36 @@
...
   <p class="lead">
-    Welcome to the Angular 2 version of the <a target="_blank" href="https://github.com/angular-ui/bootstrap">Angular UI Bootstrap</a>
-    library. This library is being built from scratch in Typescript using the Bootstrap 4 CSS framework.
+     Here we are, the Angular 2 version of the <a target="_blank" href="https://github.com/angular-ui/bootstrap">Angular UI Bootstrap</a>
+     library. This library is being built from scratch in Typescript using the Bootstrap 4 CSS framework.
   </p>
   <p class="lead">
+    To get started quickly you can follow <a [routerLink]="['/getting-started']">installation instructions</a> or just browse <a [routerLink]="['/components']">demos</a> for the existing components.
+  </p>
+  <p class="lead text-muted">
```
Comment (icfantv) [inline demo/src/app/default/default.component.html]: please remove the `text-muted` class.  i think this is ok to be black like the other two paragraphs.

**Comment by icfantv** [comment]
@benouat good stuff.  Here are my comments:
* I agree with @pkozlowski-opensource RE the logo.  The B is a little too "loud."  can you make it not so thick?  This would probably necessitate making it a little taller within the shield, too.  (Same for favicon).
* Since the rest of the text in the Jumbotron is white, I think the buttons/button text should be too.  I think the `#ccc` looks a little off with the rest of the text.
* I thought I saw it mentioned elsewhere, but I can't find it.  That footer not being pinned to the bottom (which existed previously) looks really weird with all the whitespace underneath it.

Nice work otherwise and apologies for taking so long to get to this.

**Comment by benouat** [comment]
Sorry for the lack of news here... I just was on vacation :)

---

I just updated the PR with a new commit, here is the result:

screen shot 2016-09-02 at 16 35 06-fullpage

**Pull request closed without merging**